holidays/countries/antigua_and_barbuda.py

```diff
@@ -0,0 +1,150 @@
+        * [No. 40 of 2012 Proclamation](https://laws.gov.ag/wp-content/uploads/2021/08/No.-40-of-2012-Proclamation-dated-the-27th-day-of-November-2012-Apponting-the-27th-of-December-2012-as-a-Public-Holiday-throughout-Antigua-and-Barbuda.pdf)
+        * <https://en.wikipedia.org/wiki/Public_holidays_in_Antigua_and_Barbuda>
+
+    Notes:
+        In accordance with No. 24 of 2006 Proclamation, National Heroes Day was celebrated on
+        Dec 11, 2006. In accordance with No. 40 of 2012 Proclamation, National Heroes Day was
+        celebrated on Dec 10, 2012.
+    """
+
+    country = "AG"
+    observed_label = "%s (observed)"
+    start_year = 1955
+
+    def __init__(self, *args, **kwargs):
+        ChristianHolidays.__init__(self)
+        InternationalHolidays.__init__(self)
+        StaticHolidays.__init__(self, AntiguaAndBarbudaStaticHolidays)
+        kwargs.setdefault("observed_rule", SUN_TO_NEXT_MON)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        self._add_observed(self._add_new_years_day("New Year's Day"))
+
+        # Good Friday.
+        self._add_good_friday("Good Friday")
+
+        # Easter Monday.
+        self._add_easter_monday("Easter Monday")
+
+        # Labour Day.
+        self._add_holiday_1st_mon_of_may("Labour Day")
+
+        # Whit Monday.
+        self._add_whit_monday("Whit Monday")
+
+        if self._year <= 2005:
+            # Caribbean Community (Caricom) Day.
+            self._add_holiday_1st_mon_of_jul("Caribbean Community (Caricom) Day")
+
+        # Carnival Monday.
+        self._add_holiday_1st_mon_of_aug("Carnival Monday")
+
+        if self._year >= 2006:
+            # Carnival Tuesday.
+            self._add_holiday_1_day_past_1st_mon_of_aug("Carnival Tuesday")
+
+        # Independence Day.
+        independence_day_name = "Independence Day"
+        if self._year >= 2005:
+            self._add_observed(
+                self._add_holiday_nov_1(independence_day_name), rule=SAT_SUN_TO_NEXT_MON
+            )
+        else:
+            self._add_holiday_nov_1(independence_day_name)
+
+        # National Heroes Day.
+        national_heroes_day_name = "National Heroes Day"
+        if self._year == 2006:
+            self._add_holiday_dec_11(national_heroes_day_name)
+        elif self._year == 2012:
+            self._add_holiday_dec_10(national_heroes_day_name)
+        elif 2005 <= self._year < 2014:
+            self._add_holiday_dec_9(national_heroes_day_name)
+
+        # Sir Vere Cornwall Bird SNR. Day.
+        vc_bird_day_name = "Sir Vere Cornwall Bird SNR. Day"
+        if self._year >= 2014:
+            if self._year > 2019:
+                self._add_observed(
+                    self._add_holiday_dec_9(vc_bird_day_name), rule=SAT_SUN_TO_NEXT_MON
+                )
+            else:
+                self._add_holiday_dec_9(vc_bird_day_name)
```

Comment:
```suggestion
        if self._year >= 2005:
            name = (
                # Sir Vere Cornwall Bird Snr. Day.
                "Sir Vere Cornwall Bird Snr. Day"
                if self._year >= 2014
                # National Heroes Day.
                else "National Heroes Day"
            )
            if self._year == 2006:
                self._add_holiday_dec_11(name)
            elif self._year == 2012:
                self._add_holiday_dec_10(name)
            else:
                dt = self._add_holiday_dec_9(name)
                if self._year >= 2020:
                    self._add_observed(dt, rule=SAT_SUN_TO_NEXT_MON)
```
Actually, this is one holiday that changed its name.